instagram_app/src/Components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection';
+
+function PostContainer (props) {
+    return (  
+        <div>
+         {props.data.map(eachPost => <CommentSection eachPost={eachPost} />)}
```

Comment:
I would have mapped this down into even smaller components -- Header -- Img -- Footer -- CommentSection. It would make your intentions more clear so that your code could be more readable.